source/concurrent/Thread.ooc

```diff
@@ -54,8 +54,8 @@ Thread: abstract class {
 }
 
 version (unix || apple) {
-	include pthread | (_POSIX_C_SOURCE=200809L)
-
+	// This should be in external/pthread.ooc but fails for unknown reason
+	include pthread | (_XOPEN_SOURCE=500, _POSIX_C_SOURCE=200809L)
```

Comment:
I get errors like "Cannot match `ThreadId`, nearest match is `ThreadId`" and "missing function `set(ThreadId)` in `Cell<ThreadId>`, nearest match is `set(ThreadId)`"

Go home rock, you're drunk.